admin_app/src/app/content/page.tsx

```diff
@@ -39,7 +39,7 @@ import { PageNavigation } from "./components/PageNavigation";
 import { SearchBar, SearchBarProps } from "./components/SearchBar";
 import { SearchSidebar } from "./components/SearchSidebar";
 
-const MAX_CARDS_TO_FETCH = 200;
+const MAX_CARDS_TO_FETCH = 500;
```

Comment:
Added this so that more than 200 contents can be added displayed in the frontend.